internal/backend/storage/gitfs/git.go

```diff
@@ -198,6 +198,22 @@ func (g *Git) HasStagedChanges(ctx context.Context) bool {
 	return false
 }
 
+// ListUntrackedFiles lists untracked files
+func (g *Git) ListUntrackedFiles(ctx context.Context) []string {
+	stdout, _, err := g.captureCmd(ctx, "gitLsFiles", "ls-files", ".", "--exclude-standard", "--others")
```

Comment:
Doesn't that need a little `g.fs.Path()` for the path?

Reply:
Good question! I didn't even think about this. It's been a while since I did touch this code.
But if you look inside `captureCmd` in line 118 you should see that this is being taken care of.